### ISSUE TITLE
parannuksia

### ISSUE THREAD

**Issue by SimSaladin** [TK009/loyly#8]
- kalenterin käyttäjäystävällisyys
- gallerian albumilistaus voisi olla kuvallinen
- aikaleimojen lokalisaatio puoliksi rikki
- kuvien/albumien näkyminen on epäselvää galleriassa

**Comment by KatsCyl** [comment]
Lisäisin tähän listaan vielä löylyprofiilin laajentamisen, eli että sitä voisi ehkä muuttaa, ja kuvan voisi lisätä.

**Comment by TK009** [comment]
EDIT: aikaleimat on jo toisessa issuessa

**Comment by TK009** [comment]
Oliko siis näkyvyysindikaattorit se mitä hait tolla näkymisellä?

**Comment by SimSaladin** [comment]
joo, eli mikä on julkista ja mikä ei (luultavasti, en muista enää mistä nää ehdotukset tuli alunperin)
